errors/types.go

```diff
@@ -19,8 +19,8 @@ import (
 )
 
```

Comment:
changes in this file are irrelevant. They are remnants of a previous PR and will disappear after migration to v2 code.